src/chrome/content/rules/360.cn.xml

```diff
@@ -92,7 +87,11 @@
 	<target host="shouji.360.cn" />
 	<target host="t.360.cn" />
 	<target host="t321.tianji.360.cn" />
+	<target host="wangzhan.360.cn" />
 	<target host="yunpan.360.cn" />
+	<target host="*.yunpan.360.cn" />
+		<test url="http://c2.yunpan.360.cn/" />
+		<test url="http://c12.yunpan.360.cn/" />
```

Comment:
Can you provide one more test url here?

Reply:
Any numbers from 1 to 13 IIRC. They are shutting down this service shortly, so if there are any issues regarding the yunpan.360.cn subdomain I might just remove them altogether.

Reply:
It's just because we require 3 in our style guide